models/ldndc/README.md

```diff
@@ -0,0 +1,100 @@
+A generic template for adding a new model to PEcAn
```

Comment:
You can modify the `README.md` or remove it from the package. You can do the former as you'll be updating the documentation anyway. Maybe keep things that are more factual (needs less updating) here, like some generic info, who to contact for model executable, some reference websites publications etc. (as it is explained here basically)

Reply:
Generic info added for now.